src/screens/profile/components/career-layout/career-layout.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { TouchableOpacity, View } from 'react-native';
+import { Card } from 'react-native-paper';
+
+import {
+  CalendarIcon,
+  DeleteIcon,
+  Heading,
+  PencilIcon,
+  Text,
+} from 'src/components';
+import { AppColor, HeadingLevel } from 'src/common/enums';
+
+import styles from './styles';
+
+// type CardItemProps = {
+//   startDate: Date;
+//   endDate: Date | null;
+//   position: string;
+//   company: string;
+//   title: string;
+// };
+//  type CareerLayoutProps = {
+//    cards: CardItemProps[];
+// };
+
+// const CareerLayout: React.FC<CareerLayoutProps> = () => (
+const CareerLayout = () => (
+  <View style={styles.careerWrapper}>
+    <View style={styles.divider} />
+    <View style={styles.cards}>
+      <View style={styles.careerItem}>
+        <View>
+          <View style={styles.badge} />
+          <Text style={styles.careerYear}>2023</Text>
+        </View>
+        <Card style={styles.card}>
+          <Card.Content style={styles.cardContent}>
+            <Heading level={HeadingLevel.H5} style={styles.cardTitle}>
+              Fullstack JS Developer
```

Comment:
Don't use hardcoded values. They should be passed from upper level or, at least, extracted to variables